extract_unique_sensors_01.py

```diff
@@ -0,0 +1,28 @@
+from urllib.request import urlopen
+
+sensors_data = urlopen('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+lines = sensors_data.readlines()
+
+
+def sensors(lines):
+    #global lines
```

Comment:
- Missing Docstring!
See https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit#
- Remove commented code if it doesn't add value then we do not need it!